xrally_kubernetes/service.py

```diff
@@ -69,3 +70,12 @@ def __init__(self, spec, name_generator=None, atomic_inst=None):
 
     def get_version(self):
         return version_api.VersionApi(self.api).get_code().to_dict()
+
+    @atomic.action_timer("kube.list_namespaces")
+    def list_namespaces(self):
+        """List namespaces."""
+        try:
+            self.v1_client.list_namespace()
+        except Exception:
```

Comment:
I dislike this thing. It will hide any kind of errors. even auth. do not do this. 
If there is something not "user-friendly", it should be processed (catches and re-raised a better error)